includes/before_body.tex

```diff
@@ -6,7 +6,38 @@
 \thispagestyle{empty}
 
 \begin{center}
-FIXME
+To David Flanders\\
+who taught me so much about growing and sustaining coding communities.\\
```

Comment:
```suggestion
who has taught me so much about growing and sustaining coding communities.\\
```

Reply:
I shortened it by one word to avoid an ugly line break (the last word was spilling over). We can put it back and worry about it in the edit?

Reply:
No problem - that that case you can just leave the word "has" out :smile: